librocco_receipt_printer/print_job.py

```diff
@@ -14,5 +23,52 @@ class PrintJob(Document):
     TODO: Make this more elaborate, this is just a placeholder
     """
 
+    def __str__(self):
+        str = f"<PrintJob for '{self.printer_id}'\n"
```

Comment:
The `<ObjType Foo Bar Baz>` format is typical of `__repr__` methods. The `__str__` method should return somthing user-displayable. I suggest skipping this first line.

Reply:
I made the first line say `Receipt  '<receipt-id>':` to provide some form of a heading (separating different jobs)